cirq/google/api/v1/programs.py

```diff
@@ -350,7 +350,7 @@ def _parameterized_value_from_proto_dict(message: Dict
 def _parameterized_value_to_proto_dict(param: Union[sympy.Basic, float]
                                       ) -> Dict:
     out = {}  # type: Dict
-    if isinstance(param, sympy.Symbol):
+    if isinstance(param, sympy.Basic):
```

Comment:
will the result in a formula trying to be passed in to parameter key?  for the proto stuff we only really support string symbols right now.

Reply:
Oops.  I was find-replacing too fast.

Reply:
Fixed.